contracts/mocks/Disposables/DisposableCover.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+
+import "../../abstract/MasterAwareV2.sol";
+import "../../interfaces/ICover.sol";
+
+contract DisposableCover is MasterAwareV2 {
```

Comment:
what's the reason for a DisposableCover ?

Reply:
It's used as a shortcut to add products and product types without going through governance since that is not the scope of all tests that rely on the existence of products/product types.